app/models/region_type.rb

```diff
@@ -0,0 +1,3 @@
+class RegionType < ApplicationRecord
+  validates :name, presence: true
+end
```

Comment:
Is this still needed, or does `Region#level` take care of this now?

Reply:
Oh right, this can go away.  YAGNI